From 024019be451764c7c6788165d30b0e2df85f9036 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Mon, 30 Nov 2020 01:51:09 +0100
Subject: [PATCH 341/478] usb: typec: anx7688: Rename pdo -> pkt to make it
 less confusing

Also clean make logging use base 16 command formatting, because
that's what's used in code everywhere.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 drivers/usb/typec/anx7688.c | 41 +++++++++++++++++++------------------
 1 file changed, 21 insertions(+), 20 deletions(-)

diff --git a/drivers/usb/typec/anx7688.c b/drivers/usb/typec/anx7688.c
index 4794018c9175..64e4a8c10f36 100644
--- a/drivers/usb/typec/anx7688.c
+++ b/drivers/usb/typec/anx7688.c
@@ -295,26 +295,26 @@ static int anx7688_send_ocm_message(struct anx7688 *anx7688, int cmd,
                                     const u8 *data, int data_len)
 {
         int ret = 0, i;
-        u8 pdo[32];
+        u8 pkt[32];
 
-        if (data_len > sizeof(pdo) - 3) {
+        if (data_len > sizeof(pkt) - 3) {
                 dev_dbg(anx7688->dev,
-                        "invalid ocm message length cmd=%d len=%d\n",
+                        "invalid ocm message length cmd=0x%02x len=%d\n",
                         cmd, data_len);
                 return -EINVAL;
         }
 
         // prepare pd packet
-        pdo[0] = data_len + 1;
-        pdo[1] = cmd;
+        pkt[0] = data_len + 1;
+        pkt[1] = cmd;
 	if (data_len > 0)
-		memcpy(pdo + 2, data, data_len);
-        pdo[2 + data_len] = 0;
+		memcpy(pkt + 2, data, data_len);
+        pkt[2 + data_len] = 0;
         for (i = 0; i < data_len + 2; i++)
-                pdo[data_len + 2] -= pdo[i];
+                pkt[data_len + 2] -= pkt[i];
 
-        dev_dbg(anx7688->dev, "send pd packet cmd=%d %*ph\n",
-                cmd, data_len + 3, pdo);
+        dev_dbg(anx7688->dev, "send pd packet cmd=0x%02x %*ph\n",
+                cmd, data_len + 3, pkt);
 
         ret = anx7688_tcpc_reg_read(anx7688, ANX7688_TCPC_REG_INTERFACE_SEND);
         if (ret) {
@@ -325,7 +325,7 @@ static int anx7688_send_ocm_message(struct anx7688 *anx7688, int cmd,
 
         ret = i2c_smbus_write_i2c_block_data(anx7688->client_tcpc,
                                              ANX7688_TCPC_REG_INTERFACE_SEND,
-                                             data_len + 3, pdo);
+                                             data_len + 3, pkt);
         if (ret < 0)
                 dev_err(anx7688->dev,
                         "failed to send pd packet (err=%d)\n", ret);
@@ -854,12 +854,12 @@ static int anx7688_handle_pd_message(struct anx7688* anx7688,
 
 static int anx7688_receive_msg(struct anx7688* anx7688)
 {
-        u8 pdo[32], checksum = 0;
+        u8 pkt[32], checksum = 0;
         int i, ret;
 
         ret = i2c_smbus_read_i2c_block_data(anx7688->client_tcpc,
                                             ANX7688_TCPC_REG_INTERFACE_RECV,
-                                            32, pdo);
+                                            32, pkt);
         if (ret < 0) {
                 dev_err(anx7688->dev, "failed to read pd msg\n");
                 return ret;
@@ -870,23 +870,24 @@ static int anx7688_receive_msg(struct anx7688* anx7688)
                 dev_warn(anx7688->dev, "failed to clear recv fifo\n");
         }
 
-        if (pdo[0] == 0 || pdo[0] > sizeof(pdo) - 2) {
-                dev_err(anx7688->dev, "received invalid pd message\n");
+        if (pkt[0] == 0 || pkt[0] > sizeof(pkt) - 2) {
+                dev_err(anx7688->dev, "received invalid pd message: %*ph\n",
+			(int)sizeof(pkt), pkt);
                 return -EINVAL;
         }
 
-        dev_dbg(anx7688->dev, "recv ocm message cmd=%d %*ph\n",
-                pdo[1], pdo[0] + 2, pdo);
+        dev_dbg(anx7688->dev, "recv ocm message cmd=0x%02x %*ph\n",
+                pkt[1], pkt[0] + 2, pkt);
 
-        for (i = 0; i < pdo[0] + 2; i++)
-                checksum += pdo[i];
+        for (i = 0; i < pkt[0] + 2; i++)
+                checksum += pkt[i];
 
         if (checksum != 0) {
                 dev_err(anx7688->dev, "bad checksum on received message\n");
                 return -EINVAL;
         }
 
-        anx7688_handle_pd_message(anx7688, pdo[1], pdo + 2, pdo[0] - 1);
+        anx7688_handle_pd_message(anx7688, pkt[1], pkt + 2, pkt[0] - 1);
         return 0;
 }
 
-- 
2.35.3

